src/core/templates/article.hbs

```diff
@@ -5,7 +5,7 @@
     <div class="article-header">
         {{#if displayTitle}}
         <div class="article-title">
-            <div role="heading" tabindex="0" class="article-title-inner h2"  aria-level="2">
+            <div class="article-title-inner h2" {{{a11y_attrs_heading}}}>
```

Comment:
you can remove the "hX" classes now i think. they're not used anywhere.
can we do:
```hbs
{{{a11y_attrs_heading 'article'}}}
{{{a11y_attrs_heading 1}}}
```
instead of the ``instanceof`` checks? the structure is likely to change

Reply:
To my knowledge, nothing in the vanilla theme now references the 'hX' classes as the hierarchy is governed by the ```aria-level='X'```